From 80b2423dc9a8e0c5484168f7bfe10aef38f3b9b5 Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Tue, 20 Jun 2023 10:06:57 +0200
Subject: [PATCH 6/7] update ts

---
 axe.d.ts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/axe.d.ts b/axe.d.ts
index 95997ca0..b5ae512d 100644
--- a/axe.d.ts
+++ b/axe.d.ts
@@ -164,12 +164,6 @@ declare namespace axe {
     target: string[];
     html: string;
   }
-  interface RuleLocale {
-    [key: string]: {
-      description: string;
-      help: string;
-    };
-  }
   interface CheckMessages {
     pass: string | { [key: string]: string };
     fail: string | { [key: string]: string };
@@ -180,7 +174,6 @@ declare namespace axe {
   }
   interface Locale {
     lang?: string;
-    rules?: RuleLocale;
     checks?: CheckLocale;
   }
   interface AriaAttrs {
-- 
2.37.1 (Apple Git-137.1)

