From 2eff65d9f82f0016c186c29ae21a76d5f0f61561 Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Thu, 7 Sep 2023 09:22:03 +0200
Subject: [PATCH] remove more constructHelpUrl tests

---
 test/core/base/audit.js | 60 -----------------------------------------
 1 file changed, 60 deletions(-)

diff --git a/test/core/base/audit.js b/test/core/base/audit.js
index 7160c0d..6904f35 100644
--- a/test/core/base/audit.js
+++ b/test/core/base/audit.js
@@ -290,66 +290,6 @@ describe('Audit', () => {
       assert.equal(audit.brand, 'axe');
       assert.equal(audit.application, 'thing');
     });
-    it('should call _constructHelpUrls', () => {
-      audit = new Audit();
-      audit.addRule({
-        id: 'target',
-        matches: 'function () {return "hello";}',
-        selector: 'bob'
-      });
-      assert.lengthOf(audit.rules, 1);
-      assert.equal(audit.data.rules.target, undefined);
-      audit.setBranding({
-        application: 'thing'
-      });
-      assert.deepEqual(audit.data.rules.target, {
-        helpUrl:
-          'https://dequeuniversity.com/rules/axe/' +
-          ver +
-          '/target?application=thing'
-      });
-    });
-    it('should call _constructHelpUrls even when nothing changed', () => {
-      audit = new Audit();
-      audit.addRule({
-        id: 'target',
-        matches: 'function () {return "hello";}',
-        selector: 'bob'
-      });
-      assert.lengthOf(audit.rules, 1);
-      assert.equal(audit.data.rules.target, undefined);
-      audit.setBranding(undefined);
-      assert.deepEqual(audit.data.rules.target, {
-        helpUrl:
-          'https://dequeuniversity.com/rules/axe/' +
-          ver +
-          '/target?application=axeAPI'
-      });
-    });
-    it('should not replace custom set branding', () => {
-      audit = new Audit();
-      audit.addRule({
-        id: 'target',
-        matches: 'function () {return "hello";}',
-        selector: 'bob',
-        metadata: {
-          helpUrl:
-            'https://dequeuniversity.com/rules/customer-x/' +
-            ver +
-            '/target?application=axeAPI'
-        }
-      });
-      audit.setBranding({
-        application: 'thing',
-        brand: 'other'
-      });
-      assert.equal(
-        audit.data.rules.target.helpUrl,
-        'https://dequeuniversity.com/rules/customer-x/' +
-          ver +
-          '/target?application=axeAPI'
-      );
-    });
   });
 
   describe('Audit#addRule', () => {
-- 
2.39.2 (Apple Git-143)

