From 4ea8890b1fcbf112832af8246e0808578db1c0e9 Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Thu, 10 Aug 2023 16:00:53 +0200
Subject: [PATCH] remove some pollyfills

---
 lib/core/imports/index.js | 27 ---------------------------
 1 file changed, 27 deletions(-)

diff --git a/lib/core/imports/index.js b/lib/core/imports/index.js
index f88ac87..d53acf9 100644
--- a/lib/core/imports/index.js
+++ b/lib/core/imports/index.js
@@ -4,37 +4,10 @@ import emojiRegexText from 'emoji-regex';
 import memoize from 'memoizee';
 import Color from 'colorjs.io';
 
-import es6promise from 'es6-promise';
-import { Uint32Array } from 'typedarray';
-import 'weakmap-polyfill';
-
 // prevent striping newline characters from strings (e.g. failure
 // summaries). value must be synced with build/configure.js
 doT.templateSettings.strip = false;
 
-if (!('Promise' in window)) {
-  es6promise.polyfill();
-}
-
-if (!('Uint32Array' in window)) {
-  window.Uint32Array = Uint32Array;
-}
-if (window.Uint32Array) {
-  // @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/some
-  if (!('some' in window.Uint32Array.prototype)) {
-    Object.defineProperty(window.Uint32Array.prototype, 'some', {
-      value: Array.prototype.some
-    });
-  }
-
-  if (!('reduce' in window.Uint32Array.prototype)) {
-    // @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/reduce
-    Object.defineProperty(window.Uint32Array.prototype, 'reduce', {
-      value: Array.prototype.reduce
-    });
-  }
-}
-
 /**
  * Namespace `axe.imports` which holds required external dependencies
  *
-- 
2.39.2 (Apple Git-143)

