From 9697b1ea3421e8c64b9b0fe3564ca9cba5c1296b Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Thu, 7 Sep 2023 09:35:15 +0200
Subject: [PATCH] remove setBranding test

---
 test/core/base/audit.js | 31 -------------------------------
 1 file changed, 31 deletions(-)

diff --git a/test/core/base/audit.js b/test/core/base/audit.js
index 5129651..a126106 100644
--- a/test/core/base/audit.js
+++ b/test/core/base/audit.js
@@ -103,37 +103,6 @@ describe('Audit', () => {
     });
   });
 
-  describe('Audit#setBranding', () => {
-    it('should change the brand', () => {
-      audit = new Audit();
-      assert.equal(audit.brand, 'axe');
-      assert.equal(audit.application, 'axeAPI');
-      audit.setBranding({
-        brand: 'thing'
-      });
-      assert.equal(audit.brand, 'thing');
-      assert.equal(audit.application, 'axeAPI');
-    });
-    it('should change the application', () => {
-      audit = new Audit();
-      assert.equal(audit.brand, 'axe');
-      assert.equal(audit.application, 'axeAPI');
-      audit.setBranding({
-        application: 'thing'
-      });
-      assert.equal(audit.brand, 'axe');
-      assert.equal(audit.application, 'thing');
-    });
-    it('should change the application when passed a string', () => {
-      audit = new Audit();
-      assert.equal(audit.brand, 'axe');
-      assert.equal(audit.application, 'axeAPI');
-      audit.setBranding('thing');
-      assert.equal(audit.brand, 'axe');
-      assert.equal(audit.application, 'thing');
-    });
-  });
-
   describe('Audit#addRule', () => {
     it('should override existing rule', () => {
       audit = new Audit();
-- 
2.39.2 (Apple Git-143)

