From e73c2ae76cd8e79aecf1f262bad7142caf17a22d Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Mon, 28 Oct 2024 22:04:24 +0100
Subject: [PATCH] remove branding tests

---
 test/core/base/audit.js | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/test/core/base/audit.js b/test/core/base/audit.js
index 156397c..ede8cde 100644
--- a/test/core/base/audit.js
+++ b/test/core/base/audit.js
@@ -168,35 +168,6 @@ describe('Audit', () => {
       audit.resetRulesAndChecks();
       assert.equal(audit.lang, 'en');
     });
-    it('should reset brand', () => {
-      audit = new Audit();
-      assert.equal(audit.brand, 'axe');
-      audit.setBranding({
-        brand: 'test'
-      });
-      assert.equal(audit.brand, 'test');
-      audit.resetRulesAndChecks();
-      assert.equal(audit.brand, 'axe');
-    });
-    it('should reset brand application', () => {
-      audit = new Audit();
-      assert.equal(audit.application, 'axeAPI');
-      audit.setBranding({
-        application: 'test'
-      });
-      assert.equal(audit.application, 'test');
-      audit.resetRulesAndChecks();
-      assert.equal(audit.application, 'axeAPI');
-    });
-    it('should reset brand tagExlcude', () => {
-      axe._load({});
-      assert.deepEqual(axe._audit.tagExclude, ['experimental', 'deprecated']);
-      axe.configure({
-        tagExclude: ['ninjas']
-      });
-      axe._audit.resetRulesAndChecks();
-      assert.deepEqual(axe._audit.tagExclude, ['experimental', 'deprecated']);
-    });
 
     it('should reset noHtml', () => {
       audit = new Audit();
-- 
2.39.5 (Apple Git-154)

