From 42e8a4f25952675d98456ab93abfe0a5968ef9f4 Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Thu, 7 Sep 2023 09:36:59 +0200
Subject: [PATCH] remove branding tests

---
 test/core/base/audit.js | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/test/core/base/audit.js b/test/core/base/audit.js
index a126106..ede8cde 100644
--- a/test/core/base/audit.js
+++ b/test/core/base/audit.js
@@ -168,35 +168,6 @@ describe('Audit', () => {
       audit.resetRulesAndChecks();
       assert.equal(audit.lang, 'en');
     });
-    it('should reset brand', () => {
-      audit = new Audit();
-      assert.equal(audit.brand, 'axe');
-      audit.setBranding({
-        brand: 'test'
-      });
-      assert.equal(audit.brand, 'test');
-      audit.resetRulesAndChecks();
-      assert.equal(audit.brand, 'axe');
-    });
-    it('should reset brand application', () => {
-      audit = new Audit();
-      assert.equal(audit.application, 'axeAPI');
-      audit.setBranding({
-        application: 'test'
-      });
-      assert.equal(audit.application, 'test');
-      audit.resetRulesAndChecks();
-      assert.equal(audit.application, 'axeAPI');
-    });
-    it('should reset brand tagExlcude', () => {
-      axe._load({});
-      assert.deepEqual(axe._audit.tagExclude, ['experimental']);
-      axe.configure({
-        tagExclude: ['ninjas']
-      });
-      axe._audit.resetRulesAndChecks();
-      assert.deepEqual(axe._audit.tagExclude, ['experimental']);
-    });
 
     it('should reset noHtml', () => {
       audit = new Audit();
-- 
2.39.2 (Apple Git-143)

