From e072fbaa639673943597d668978b12ed6587c021 Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Thu, 10 Aug 2023 16:23:11 +0200
Subject: [PATCH] remove failure message from axe.d.ts

---
diff --git a/axe.d.ts b/axe.d.ts
index 41b282d..f0ff2ae 100644
--- a/axe.d.ts
+++ b/axe.d.ts
@@ -153,7 +153,6 @@ declare namespace axe {
     any: CheckResult[];
     all: CheckResult[];
     none: CheckResult[];
-    failureSummary?: string;
     element?: HTMLElement;
   }
   interface CheckResult {
-- 
2.39.2 (Apple Git-143)

