From 3515aab153cec71dd6ffd23f88588e2f888903db Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Mon, 28 Oct 2024 22:09:32 +0100
Subject: [PATCH] remove unused fields from getRules

---
 lib/core/public/get-rules.js | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lib/core/public/get-rules.js b/lib/core/public/get-rules.js
index fa23a3b..9d3e6f0 100644
--- a/lib/core/public/get-rules.js
+++ b/lib/core/public/get-rules.js
@@ -13,14 +13,9 @@ function getRules(tags) {
         }).length;
       });
 
-  const ruleData = axe._audit.data.rules || {};
   return matchingRules.map(matchingRule => {
-    const rd = ruleData[matchingRule.id] || {};
     return {
       ruleId: matchingRule.id,
-      description: rd.description,
-      help: rd.help,
-      helpUrl: rd.helpUrl,
       tags: matchingRule.tags,
       actIds: matchingRule.actIds
     };
-- 
2.39.5 (Apple Git-154)

