From 7353de5cd93914e752d19d77cf3e57bcb07a23df Mon Sep 17 00:00:00 2001
From: ArjanSchouten <arjan.schouten@solureal.com>
Date: Mon, 28 Oct 2024 22:27:12 +0100
Subject: [PATCH] fix cannot access axe before initialization

---
 lib/core/index.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/core/index.js b/lib/core/index.js
index 2d15d52..e2e70c9 100644
--- a/lib/core/index.js
+++ b/lib/core/index.js
@@ -2,7 +2,7 @@
 /*global axeFunction, module, define */
 // exported namespace for axe
 /*eslint no-use-before-define: 0, no-unused-vars: 0*/
-const axe = axe || {};
+const axe = {};
 axe.version = '<%= pkg.version %>';
 
 if (typeof define === 'function' && define.amd) {
-- 
2.39.5 (Apple Git-154)

